### ISSUE TITLE
Automate unit tests running on pushes to repository

### ISSUE THREAD

**Issue by maddiebeagley** [SE701ProjectGroup4/Flatmate-Management-System#30]
**User story**
As a developer, I want to ensure all code quality is consistent with unit tests when I push any changes to the repository. 

**Acceptance criteria**
- All unit tests are run when commits are pushed to the repository
- I can view any tests that have failed so I can fix them
- Branches cannot be merged into master if any tests are failing

**Blocking**
Currently, this issue is blocked by #29 as we cannot verify if our automated script works until we have tests to run.

**Comment by DragonMeme** [comment]
I also assume that both the front-end and the back-end unit tests to be run on push? 
Since this is technically 2 projects in 1 repository.

**Comment by jhsi517** [comment]
Closing this ticket since it was marked as done.
